### DISCUSSION TITLE
How to use Vite's legacy feature for low-version browser compatibility in Nuxt 3

### DISCUSSION THREAD

**Discussion by ivy-zyy** [nuxt/nuxt#23923] [Questions]
I encountered an issue while using Nuxt 3 with Vite for low-version browser compatibility.
I observed in previous issues that when using the @vitejs/plugin-legacy library for bundling, the generated index.html file should include both <script module> and <script nomodule> sections to cater to different browser requirements. 
However, in my case, the index.html does not contain the <script nomodule> section. I also attempted to use the hooks: 'vite:extendConfig', but the result remained the same.

Here is my configuration:
```typescript
import legacy from '@vitejs/plugin-legacy';
 export default defineNuxtConfig({
 // method 1
 hooks: {
    'vite:extendConfig'(clientConfig, { isClient, isServer }) {
      if (isClient) {
        clientConfig.plugins!.push(
          legacy({
            targets: ['ie >= 11'],
            renderLegacyChunks: true,
            modernPolyfills: true
          })
        );
      }
    }
  },
 // method 2
 vite: {
 plugins: [
       legacy({
         targets: ['ie >= 11'],
         modernPolyfills: true
       })
   ]
 }
;
``` 
Thank you for help!

**Comment by 275923543** [comment]
nuxt3-vite-legacy
